buildSrc/src/main/kotlin/toolkit-intellij-subplugin.gradle.kts

```diff
@@ -0,0 +1,147 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import org.jetbrains.intellij.tasks.DownloadRobotServerPluginTask
+import org.jetbrains.intellij.tasks.RunIdeForUiTestTask
+import org.jetbrains.intellij.tasks.RunIdeTask
+import software.aws.toolkits.gradle.IdeVersions
+import software.aws.toolkits.gradle.ciOnly
+import software.aws.toolkits.gradle.findFolders
+import software.aws.toolkits.gradle.intellij
+import software.aws.toolkits.gradle.intellij.ToolkitIntelliJExtension
+import software.aws.toolkits.gradle.intellij.ToolkitIntelliJExtension.IdeFlavor
+import java.time.Instant
+
+val toolkitIntelliJ = project.extensions.create<ToolkitIntelliJExtension>("intellijToolkit")
+
+val ideProfile = IdeVersions.ideProfile(project)
+val remoteRobotPort: String by project
+val remoteRobotVersion: String by project
+
+plugins {
+    id("toolkit-kotlin-conventions")
+    id("toolkit-testing")
+    id("org.jetbrains.intellij")
+}
+
+// Run after the project has been evaluated so that the extension (intellijToolkit) has been configured
+afterEvaluate {
+    val flavor = toolkitIntelliJ.ideFlavor.get()
+    val productProfile = when (flavor) {
+        IdeFlavor.IC -> ideProfile.community
+        IdeFlavor.IU -> ideProfile.ultimate
+        IdeFlavor.RD -> ideProfile.rider
+    }
+
+    intellij {
+        pluginName = "aws-toolkit-jetbrains"
+        version = productProfile.sdkVersion
+
+        setPlugins(*productProfile.plugins)
+
+        downloadSources = flavor != IdeFlavor.IC
+        instrumentCode = flavor != IdeFlavor.RD
+    }
+
+    tasks.jar {
+        archiveBaseName.set("aws-toolkit-jetbrains-$flavor")
+    }
+
+    tasks.patchPluginXml {
+        setSinceBuild(ideProfile.sinceVersion)
+        setUntilBuild(ideProfile.untilVersion)
+    }
+
+    tasks.buildSearchableOptions {
+        enabled = false
```

Comment:
Add a comment that the build fails if this is enabled?

Reply:
added